README.md

```diff
@@ -21,17 +32,23 @@ deployer also serves as a CI agent.
         aws_access_key_id=...
         aws_secret_access_key=...
 
-5.  Check `scripts/bootstrap.sh` and modify variables if needed.
-6.  Enter `nix-shell`. This may take several minutes as nixpkgs is being
-    fetched.
-7.  Run `scripts/bootstrap.sh`.
-8.  SSH into the domain name printed by the script.
-9.  Run `nixops deploy -d deployer` to finish deployment. This may again take
-    several minutes as nixpkgs is being fetched.
+5.  Review `scripts/bootstrap.sh` and modify the variables if needed.
+6.  Enter `nix-shell`. This may take several minutes without any output
+    as nixpkgs is being fetched.
+7.  Run `./scripts/bootstrap.sh`.
+8.  If you were using a temporary instance for bootstrapping, at this point
```

Comment:
I recommend discarding it only after you've made sure the deployer can deploy itself. Otherwise, fixing it or cleaning up can be a real pain.